ethereum/spec/src/main/java/tech/pegasys/teku/spec/datastructures/operations/versions/bellatrix/BeaconPreparableProposer.java

```diff
@@ -60,4 +84,33 @@ public String toString() {
         .add("feeRecipient", feeRecipient)
         .toString();
   }
+
+  public static DeserializableTypeDefinition<BeaconPreparableProposer> getJsonTypeDefinition() {
+    return BEACON_PREPARABLE_PROPOSER_TYPE;
+  }
```

Comment:
Hmm, this feels like a change to the pattern of where type definitions are defined. Normally they're separate from the actual data class.  I think it might be blurring the lines between internal and external representations... Not a strong opinion though.

There is some confusion caused by this class having wound up in spec/datastructures/operations when it's not actually an operation and not even really part of the beacon chain spec. It's really just an internal class we created because we needed it and it happens to have the same data as the REST API endpoint needs.